main.go

```diff
@@ -282,7 +284,13 @@ func (ctx *httpContext) OnHttpResponseBody(bodySize int, endOfStream bool) types
 		return types.ActionContinue
 	}
 	if interruption != nil {
-		// TODO(M4tteoP): Address response body interruption logic after https://github.com/corazawaf/coraza-proxy-wasm/issues/26
+		// TODO(M4tteoP): Update response body interruption logic after https://github.com/corazawaf/coraza-proxy-wasm/issues/26
+		// Currently returns a body filled with null bytes that replaces the sensitive data potentially leaked
+		err = proxywasm.ReplaceHttpResponseBody(bytes.Repeat([]byte("\x00"), ctx.responseBodySize))
```

Comment:
We definitively need to output a log whenever we are replacing user data. I guess warning is the right level.